src/app/base/components/dropdown/dropdown-helper.ts

```diff
@@ -0,0 +1,38 @@
+import { FormArray } from "@angular/forms";
+
+/**
+ * Optionset for kern dropdown
+ * @optionArray {string[] | FormArray[] | any[]} - Can be any array. string array is just used, FormArray or any object array tries to use "name" property then "text" last first string property
+ * @buttonCaption {string, optional} - used as caption for the button, if not given the first / current value is used
+ * @valuePropertyPath {string, optional} - if undefined option text is returned, else (e.g. name.tmp.xyz) the path is split and used to access the object property
+ * @keepDropdownOpen {boolean, optional} - stops the event propagation of the click event and therfore keeps the menu open
+ * @buttonTooltip {string, optional} - adds a tooltip if defined
+ * @isDisabled {boolean, optional} - disables the dropdown
+ * @isOptionDisabled {boolean[], optional} - disables the dropdown option (needs to be the exact same length as the optionArray)
+ * @optionIcons {string[], optional} - displays a predfined icon if set for the index (needs to be the exact same length as the optionArray)
+ * @hasCheckboxes {boolean, optional} - helper for checkbox like dropdowns (e.g. data browser)
+ * @buttonVersion {string, optional} - defaults to 'default' (button with a caption text), '...', 'userIcon'
+ * @avatarUri {string, optional} - link to the avatar image for logged user
+ * @prefix {string, optional} - prefix to the main name in the option
+ * @postfix {string, optional} - postfix to the main name in the option
+ * @buttonHasBlueStyle {boolean, optional} - flag if the button has the blue style
```

Comment:
this isn't really clear to me, what is the "blue" style?

Reply:
yes but is it a blue tint or is it a "special" option like hasCheckboxes that has custom logic only meant for that specific usecase?

Reply:
also doesnt seem to work as this is a different color than on the dev app:
current:
![image](https://user-images.githubusercontent.com/91723236/183675927-de07eea6-7203-49e7-adb8-5c8a83b527d0.png)

dev app:
![image](https://user-images.githubusercontent.com/91723236/183676033-5efd5eb9-cd55-4d7b-8401-74cfd966e44f.png)


also I think the size is different (test size)

Reply:
No, it is for all buttons that we want to have this style.

Reply:
Yes, I saw that but I thought all drodowns should have the same style, if this one is so different that the others does it make sense to be replaced with the shared component at all? Or we can add condition to use specific characteristics for this dropdown.

Reply:
i think i'd go with options for 
1. hovertint or hovercolor (gray = default, blue = this blue & hexcode for everything else)
2. text color (default = black & hexcode for everything else)
3. text hover color (default = white & hexcode for everything else)
4. text size (default = text-sm (class names possible))